packages/config/index.d.ts

```diff
@@ -102,6 +102,7 @@ export interface StackableInterface {
   setConnectionStatus?: (status: string) => Promise<void>
   setOpenapiSchema?: (schema: object) => Promise<void>
   setGraphqlSchema?: (schema: string) => Promise<void>
+  setCustomHealthCheck?: (healthCheck: Function) => Promise<void>
```

Comment:
@ShogunPanda is it correct? Do I miss any other places to set types?

Reply:
Nope, it should look fine